app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
github is not just an alternative place to bugzilla to voice your concerns, this is where you get involved yourself to help. The (already existing) bug is enough to make maintainers aware of the issue, if that's what you want.